cli/pkg/workspace/init.go

```diff
@@ -3,22 +3,25 @@ package workspace
 import (
 	"cli/pkg/workspace/template"
 	"fmt"
-	"github.com/spf13/viper"
 	"os"
 	"path/filepath"
+
+	"github.com/spf13/viper"
 )
 
-func Init(path string) ConfigDir {
+func Init(path string, log bool) ConfigDir {
```

Comment:
It's a bit of a bad practice to pass boolean flags to methods, especially if the flag only has one use case. Here: "Don't log a failure for the `airy version` command".

The idiomatic way would be to return an error `func Init(path string, log bool) (ConfigDir, error)`. Then the function users can decide what to do with that error (log or ignore)

Reply:
The update now works with the error return. Let me know if this is the correct way!